### DISCUSSION TITLE
SSO from SPA

### DISCUSSION THREAD

**Discussion by jakubkoje** [jlguenego/node-expose-sspi#73] [Q&A] [answered]
Hello, is it possible to use this library to authenticate API calls from Vue application?

**Comment by jlguenego** [comment, marked as answer]
yes. There is an example. https://github.com/jlguenego/vue-sso-example

On Sat, Feb 20, 2021 at 9:52 PM Jakub Jelínek <notifications@github.com>
wrote:

> Hello, is it possible to use this library to authenticate API calls from
> Vue application?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jlguenego/node-expose-sspi/discussions/73>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAVV6TSZABEBCHY534ESM3TTAAOHVANCNFSM4X6HQV2A>
> .
>


-- 
Jean-Louis GUENEGO
Tel : +33 6 12 19 81 48
mail: jlguenego@gmail.com
